Makefile

```diff
@@ -17,7 +17,9 @@ serve:
 	mvn clean verify
 	mvn -P cargo.run,without-autoexport
 
-test%:
+test:
```

Comment:
Does this work differently to the single test% matcher you had before?

Reply:
unfortunately yes. `%` is like `.+`